client/blocks/qr-code-login/index.jsx

```diff
@@ -0,0 +1,198 @@
+import { getTracksAnonymousUserId } from '@automattic/calypso-analytics';
```

Comment:
I'm curious to find out more about the rationale of using this. Could you elaborate?

Reply:
The idea here is to use something that is randomly generated and and is different between the different browsers. 

See p3btAN-1Jr-p2